src/app/public/modules/phone-field/phone-field.component.spec.ts

```diff
@@ -0,0 +1,849 @@
+import {
+  ComponentFixture,
+  TestBed,
+  fakeAsync,
+  tick
+} from '@angular/core/testing';
+
+import {
+  FormsModule,
+  NgModel,
+  ReactiveFormsModule
+} from '@angular/forms';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  PhoneFieldTestComponent
+} from './fixtures/phone-field.component.fixture';
+
+import {
+  PhoneFieldReactiveTestComponent
+} from './fixtures/phone-field-reactive.component.fixture';
+
+import {
+  SkyPhoneFieldModule
+} from './phone-field.module';
+
+fdescribe('Phone Field Component', () => {
```

Comment:
`fdescribe`

Reply:
Oof. I'm the king at leaving those in.... fixed.